Source/Parallelization/GuardCellManager.cpp

```diff
@@ -232,6 +232,10 @@ guardCellManager::Init (
     ng_alloc_F.max( ng_FieldSolverF );
     ng_alloc_G.max( ng_FieldSolverG );
 
+    if (do_moving_window && maxwell_solver_id == MaxwellSolverAlgo::PSATD) {
+        ng_afterPushPSATD = ng_alloc_EB;
+    }
```

Comment:
Just a note for @RemiLehe: because of these lines here, I think we should not expect a performance improvement for simulations with a moving window (e.g. milestone runs, If I'm not mistaken).

Reply:
Indeed, this PR does not change anything for simulations with a moving window.